File: backend/recipeapp/models/commentModel.go
package models

Type Comment struct {
	ID          uuid.UUID `gorm:"type:char(36);primary_key;"`
	UserId      uuid.UUID `gorm:"type:char(36);"`
	RecipeId    uuid.UUID `gorm:"type:char(36);"`
	Date        string
	Description string
	Score       int
}

File: frontend/recipeapp/lib/models/comment_model.dart
import 'package:cookingenial/models/user_model.dart';

class CommentModel {
  final String id;
  final String description;
  final int score;
  final UserModel user;
  final DateTime date;

  CommentModel({
    required this.id,
    required this.description,
    required this.score,
    required this.user,
    required this.date,
  });
}
